core/new-gui/src/app/workspace/component/operator-panel/operator-panel.component.ts

```diff
@@ -61,6 +79,55 @@ export class OperatorPanelComponent implements OnInit {
     this.operatorSchemaList = operatorMetadata.operators;
     this.groupNamesOrdered = getGroupNamesSorted(operatorMetadata.groups);
     this.operatorGroupMap = getOperatorGroupMap(operatorMetadata);
+
+
+
+    this.filteredOptions = this.myControl.valueChanges
+      .map(value => this._filter(value));
+    }
+
+  private _filter(value: string): string[] {
+
+    type userFriendlyNamesType = {
+      title: string;
+    };
+
+    const userFriendlyNames = this.operatorSchemaList.map(value => value.additionalMetadata.userFriendlyName);
+
+    const new_userFriendlyNames : userFriendlyNamesType[] = []
+    for (let entry of userFriendlyNames) {
+      new_userFriendlyNames.push({'title':entry})
+    }
```

Comment:
use `.forEach()` method provided in the newest JavaScript instead of `for-loop`.